buildSrc/src/main/java/Dependencies.kt

```diff
@@ -61,7 +62,7 @@ object Dependencies {
     const val androidx_lifecycle_extensions = "androidx.lifecycle:lifecycle-extensions:${Versions.AndroidX.lifecycle}"
 
     const val google_material = "com.google.android.material:material:${Versions.Google.material}"
-
+    const val google_accompanist_insets_ui = "com.google.accompanist:accompanist-insets-ui:${Versions.Google.accompanist}"
```

Comment:
Is this strictly needed? Just curious whether we require `accompanist` and whether we want to "officially" make it part of our compose tool set (CC-ing some folks for opinions, @jonalmeida @Mugurell @gabrielluong @mcarare)

Reply:
This accompanist lib was needed to ensure the `Scaffold` with a toolbar will be placed below the status bar.
This was needed because the`MainActivity` which shows this fragment [is set as fullscreen](https://github.com/mozilla-mobile/focus-android/blob/67129f722a3717e4b67e542e7fcb82c1de70f877/app/src/main/java/org/mozilla/focus/activity/MainActivity.kt#L68) .

Reply:
I do have `accompanist` as part of my toolset for my Top Sites rewrite on Fenix because I needed the ViewPager equivalent. I don't know much about what about Android insets to provide much opinion on this particular dependency.

Reply:
Okay, if it will be used for Fenix too, then I guess this is already decided. :) 👍